suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -185,3 +185,13 @@ export const selectAccountsAmountPerSymbol = createSelector(
             A.length,
         ),
 );
+
+export const selectImportedAccountSymbols = createSelector(
```

Comment:
It's not related to imported accounts, it will work for all accounts. Consider better name like `selectSymbolsUsedInAccounts` or just `selectAccountsSymbols`

Reply:
https://github.com/trezor/trezor-suite/pull/6991/commits/ff586ec5e32f4d479607d13122fa0c03ab633a55